app/src/main/res/values/strings.xml

```diff
@@ -4,4 +4,5 @@
     <string name="image_contentDescription">image</string>
     <string name="miles_per_hour">miles/hour</string>
     <string name="action_refresh">Refresh</string>
+    <string name="search">search</string>
```

Comment:
its good practice if we are naming a label text to add `_text` at the end of the string name. (e.g <string name="search_text">Search</string>

and always check the label if the name need to lowercase or camel case

Reply:
or if its title like in the dialog you can update it something like this `<string name="search_title">Search</string>`

Reply:
its depends on the usage